fresh.py

```diff
@@ -340,7 +342,14 @@ def main():
             addSpotifyTrack(fresh, threshold, includeAlbums, verbose, sub, tracks)
 
         else:
+            otherDomainList = ['youtu.be','youtube.com','soundcloud.com']
             # handle non-spotify posts
+            if sub.domain in otherDomainList and verbose:
```

Comment:
move this block down a few lines, inside the `if track_url:` condition. it doesnt make sense where it currently is because that track may not end up getting added to the playlist.

Reply:
Ah, I was under the impression that you wanted it to be printed out regardless of it being able to be added to the spotify playlist or not.